tutorials/weaviate_llama_index_search_and_retrieval.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>This is re-downloading the chatbot data from cloud storage and storing it in a storage context that has its own in-memory vector store (not backed by Weaviate). Can you verify that the query engine is using the Weaviate vector and not the in-memory vector store in this case?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Yes it does replace the vector store and uses weaviate </p>